rllib/algorithms/algorithm_config.py

```diff
@@ -897,7 +897,11 @@ def build_env_to_module_connector(self, env):
             if self.is_multi_agent():
                 pipeline.append(
                     AgentToModuleMapping(
-                        modules=set(self.policies),
+                        module_specs=(
+                            self.rl_module_spec.module_specs
+                            if isinstance(self.rl_module_spec, MultiAgentRLModuleSpec)
+                            else set(self.policies)
```

Comment:
Great point. We need to unify this soon and fully adapt to the new stack terminology. Some ideas:
* Have user explicitly enable multi-agent (otherwise, will error if multi-agent components are used).
* `config.multi_agent(policies)` should no longer be necessary (already kind of replaced by `config.rl_module`)
* policy_mapping_fn -> agent_to_module_mapping_fn
* etc..